app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/BaseFragment.java

```diff
@@ -30,34 +31,78 @@ public BaseFragment() {
     }
 
     @Override
+    public void onAttach(Context context) {
```

Comment:
Line 28 - Maybe this is the solution (Couldn't mark this line - out of the review section).